trimChamber.py

```diff
@@ -20,10 +20,13 @@
 parser.add_option("--vt1", type="int", dest="vt1",
                   help="VThreshold1 DAC value for all VFATs", metavar="vt1", default=100)
 
-
-uhal.setLogLevelTo( uhal.LogLevel.WARNING )
 (options, args) = parser.parse_args()
 
+if options.debug:
+    uhal.setLogLevelTo( uhal.LogLevel.DEBUG )
```

Comment:
there is pretty much no case where when debugging a tool i'm interested in seeing the `uhal::LogLevel::DEBUG` messages, `uhal::LogLevel::INFO` should be sufficient